### PR TITLE
remove old validations

### LINKED ISSUE #13: Remove require validation
Validation for input plugins should not include any validation to check if an input has content in it as that should instead be covered by general validation when the plugin is used in forms.

``` gherkin
Feature: Extraneous Validation
  As a core developer
  I want validation free from a check for content
  So that validation for whether an input is required is controlled by general form validation.

  Scenario: Remove validation
    Given that controlling whether an input should be required is a function of our forms
    When testing whether an input is required or not
    Then that test should included in the validation for this plugin.
```

---

**Extraneous Validation**
- [x] Remove validation

### PR DESCRIPTION
**Pull request by scottnath**
Removed old validation and settings

---

Resolves #13 

`DCO 1.1 Signed-off-by: Scott Nath <github@scottnath.com>`

### CODE REVIEW

**Comment by scottnath** [comment]
@Snugug the addition of the browser-script test is the best we can get until we have proper DOM script testing.

File: lib/validation.js [hunk 1]
```diff
@@ -19,10 +19,6 @@
  * @module selectValidation
  */
 
-module.exports = function selectValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
-  }
-
+module.exports = function selectValidation() {
```
Comment (Snugug) [inline lib/validation.js:22]: Should there be validation here? If the selects are related, should there be some validation enforcing their relationship?

Comment (scottnath) [reply lib/validation.js:22]: @Snugug This is a very basic input plugin with very basic javascript. Originally created to test javascript injection. No stories currently exist to expand on this functionality - when they do we should def grow this plugin.

Comment (Snugug) [reply lib/validation.js:22]: ok

**Pull request merged**